knowage-vue/src/modules/managers/businessModelCatalogue/metaweb/businessModel/tabs/metawebAttributesTab/dialogs/metawebAttributeDetail/MetawebAttributeDetailDialog.vue

```diff
@@ -0,0 +1,284 @@
+                    <span class="p-float-label">
+                        <Dropdown class="kn-material-input" v-model="properties['structural.dateformat'].value" :options="properties['structural.dateformat'].propertyType.admissibleValues" @change="updateAttribute('structural.dateformat')">
+                            <template #option="slotProps">
+                                <span>{{ getFormattedDate(new Date(), slotProps.option) }}</span>
+                            </template>
+                            <template #value="slotProps">
+                                <span>{{ getFormattedDate(new Date(), slotProps.value) }}</span>
+                            </template>
+                        </Dropdown>
+                        <label class="kn-material-input-label"> {{ properties['structural.dateformat'].propertyType.name }}</label>
+                        <small>{{ properties['structural.datatype'].value === 'TIME' ? properties['structural.format'].propertyType.description : properties['structural.dateformat'].propertyType.description }}</small>
+                    </span>
+                </div>
+
+                <div class="p-field p-col-12 p-md-6 p-mt-2" v-if="['INTEGER', 'DOUBLE', 'DECIMAL', 'BIGINT', 'FLOAT'].indexOf(properties['structural.datatype'].value) !== -1">
+                    <span class="p-float-label">
+                        <Dropdown class="kn-material-input" v-model="properties['structural.format'].value" :options="properties['structural.format'].propertyType.admissibleValues" @change="updateAttribute('structural.format')" />
+                        <label class="kn-material-input-label"> {{ properties['structural.format'].propertyType.name }}</label>
+                        <small>{{ properties['structural.format'].propertyType.description }}</small>
+                    </span>
+                </div>
+
+                <div class="p-field p-col-12 p-md-6 p-mt-2">
+                    <span class="p-float-label">
+                        <InputText class="kn-material-input" v-model.trim="properties['structural.customFunction'].value" @change="updateAttribute('structural.customFunction')" />
+                        <label class="kn-material-input-label"> {{ properties['structural.customFunction'].propertyType.name }}</label>
+                        <small>{{ properties['structural.customFunction'].propertyType.description }}</small>
+                    </span>
+                </div>
+            </div>
+        </div>
+
+        <template #footer>
+            <Button class="kn-button kn-button--primary" @click="closeDialog"> {{ $t('common.cancel') }}</Button>
+            <Button class="kn-button kn-button--primary" @click="saveAttribute"> {{ $t('common.save') }}</Button>
+        </template>
+    </Dialog>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue'
+import { iBusinessModelColumn } from '../../../../../Metaweb'
```

Comment:
will it be better to use the @ src? just in case the folder will change position

Reply:
I will make the correction now.